src/converters/piral-cycle/README.md

```diff
@@ -0,0 +1,97 @@
+[![Piral Logo](https://github.com/smapiot/piral/raw/master/docs/assets/logo.png)](https://piral.io)
```

Comment:
Does Cycle.js have a nice logo?

We could add it to `src/pages/docs/src/assets/extensions` as a png (cycle.png).

Reply:
There is this one: https://cycle.js.org/img/cyclejs_logo.svg
We'd just have to convert it to .png.

Reply:
Alright, done!